Makefile.am

```diff
@@ -28,4 +28,4 @@ libtool: $(LIBTOOL_DEPS)
 # Needed by autoconf because we use README.md instead of README.
 # See http://stackoverflow.com/q/15013672/
 README: README.md
```

Comment:
Autoconf and Linux distributions require just README (without .md, .txt or other extension).